backend/tests/regression/answer_quality/run_eval_pipeline.py

```diff
@@ -35,9 +35,7 @@ def main() -> None:
         config.use_cloud_gpu,
         config.llm,
     )
-    relari_output_folder_path = manage_data_directories(
-        run_suffix, config.output_folder, config.use_cloud_gpu
-    )
+    manage_data_directories(run_suffix, config.output_folder, config.use_cloud_gpu)
```

Comment:
this file is all just input changes for the other changes made in the other files